qa/travis.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -ev
```

Comment:
Suggest including `set -u` (treat unset variables as an error).

Reply:
Leaving this as future todo as it causes build failures. I suspect this is because the build matrix in the yaml defines some things some of the times, but not for all builds.